### PR TITLE
Support aliased tables and case insensitive table names

### PR DESCRIPTION
**Pull request by eac**
@osheroff hola

### CODE REVIEW

File: lib/shiba/parsers/mysql_select_fields.rb [hunk 1]
```diff
@@ -66,6 +67,18 @@ def parse_fields
 
           sc.scan(/,/)
         end
+
+        # resolve table aliases
+        if sc.scan(/ `.*?`\.`(.*?)` `(.*?)`/)
```
Comment (osheroff) [inline lib/shiba/parsers/mysql_select_fields.rb]: this syntax can also be `table AS alias`, let's support that too

Comment (osheroff) [reply lib/shiba/parsers/mysql_select_fields.rb]: other stuff:

- tables can be not quoted
- only handles a single table

If we're gonna bother, we should go a little further...

Comment (eac) [reply lib/shiba/parsers/mysql_select_fields.rb]: This parser only parses output from show warnings, which only has this format (e.g. no as, always quoted)

Comment (osheroff) [reply lib/shiba/parsers/mysql_select_fields.rb]: roger

**Pull request merged**